proposals/simd/NewOpcodes.md

```diff
@@ -124,9 +145,13 @@
 | f32x4.pmin      | 0xea   | f64x2.pmin      | 0xf6   |
 | f32x4.pmax      | 0xeb   | f64x2.pmax      | 0xf7   |
 
-| Conversion Op           | opcode |
-| ----------------------- | ------ |
-| i32x4.trunc_sat_f32x4_s | 0xf8   |
-| i32x4.trunc_sat_f32x4_u | 0xf9   |
-| f32x4.convert_i32x4_s   | 0xfa   |
-| f32x4.convert_i32x4_u   | 0xfb   |
+| Conversion Op                | opcode |
+| ---------------------------- | ------ |
+| i32x4.trunc_sat_f32x4_s      | 0xf8   |
+| i32x4.trunc_sat_f32x4_u      | 0xf9   |
+| f32x4.convert_i32x4_s        | 0xfa   |
+| f32x4.convert_i32x4_u        | 0xfb   |
+| f64x2.convert_low_i32x4_s    | 0xfc   |
```

Comment:
Should we flip f64x2.convert with i32x4.trunc_sat, to match the ordering we have above? trunc_sat first, then convert. No big deal though.